src/contracts/BMath.sol

```diff
@@ -13,259 +13,249 @@
 
 pragma solidity 0.8.23;
 
-import "./BNum.sol";
+import './BNum.sol';
 
 contract BMath is BBronze, BConst, BNum {
-    /**********************************************************************************************
-    // calcSpotPrice                                                                             //
-    // sP = spotPrice                                                                            //
-    // bI = tokenBalanceIn                ( bI / wI )         1                                  //
-    // bO = tokenBalanceOut         sP =  -----------  *  ----------                             //
-    // wI = tokenWeightIn                 ( bO / wO )     ( 1 - sF )                             //
-    // wO = tokenWeightOut                                                                       //
-    // sF = swapFee                                                                              //
-    **********************************************************************************************/
-    function calcSpotPrice(
-        uint tokenBalanceIn,
-        uint tokenWeightIn,
-        uint tokenBalanceOut,
-        uint tokenWeightOut,
-        uint swapFee
-    )
-        public pure
-        returns (uint spotPrice)
-    {
-        uint numer = bdiv(tokenBalanceIn, tokenWeightIn);
-        uint denom = bdiv(tokenBalanceOut, tokenWeightOut);
-        uint ratio = bdiv(numer, denom);
-        uint scale = bdiv(BONE, bsub(BONE, swapFee));
-        return  (spotPrice = bmul(ratio, scale));
-    }
+  /**
+   *
+   *   // calcSpotPrice                                                                             //
+   *   // sP = spotPrice                                                                            //
+   *   // bI = tokenBalanceIn                ( bI / wI )         1                                  //
+   *   // bO = tokenBalanceOut         sP =  -----------  *  ----------                             //
+   *   // wI = tokenWeightIn                 ( bO / wO )     ( 1 - sF )                             //
+   *   // wO = tokenWeightOut                                                                       //
```

Comment:
hmm watch out in this and all multiline comments, now we have both `/**` and `//`, plz remove the latter